library/common/extensions/key_value/platform/config.cc

```diff
@@ -63,8 +63,8 @@ PlatformKeyValueStore::PlatformKeyValueStore(Event::Dispatcher& dispatcher,
 void PlatformKeyValueStore::flush() {
   std::string output;
   for (const auto& it : store_) {
-    absl::StrAppend(&output, it.first.length(), "\n", it.first, it.second.length(), "\n",
-                    it.second);
+    absl::StrAppend(&output, it.first.length(), "\n", it.first, it.second.first.length(), "\n",
+                    it.second.first);
```

Comment:
Drive-by: This has become a bit hard to read. How about:

```
  for (const auto& [key, value] : store_) {
    absl::StrAppend(&output, key.length(), "\n", key, value.length(), "\n", value);
```

Reply:
Yes that's certainly better, thanks for the tip!